lighthouse-core/audits/maskable-icon.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const ManifestValues = require('../computed/manifest-values.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on if the manifest contains a maskable icon. This descriptive title is shown to users when the manifest does contains at least one maskable icon. */
+  title: 'Manifest has a maskable icon',
+  /** Title of a Lighthouse audit that provides detial on if the manifest contains a maskable icon. this descriptive title is shown to users when the manifest contains no icons that are maskable. */
+  failureTitle: 'Manifest doesn\'t have a maskable icon',
+  /** Description of a Lighthouse audit that tells the user why they their manifest should have at least one maskable icon. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'A maskable icon ensures that the icon fills the entire shape ' +
```

Comment:
```suggestion
  description: 'A maskable icon ensures that the image fills the entire shape without being letterboxed ' +
```

maybe? Idk, `shape` alone just feels a little ambiguous outside the context of the sentence in the article that was already discussing the icon grid.